tests/test_encode_decode.py

```diff
@@ -113,3 +113,27 @@ def test_querystringsafe_base64_decode_handles_unicode():
     assert querystringsafe_base64.decode(base64_str) == decoded_str
     assert querystringsafe_base64.decode(base64dotted_unicode) == decoded_str
     assert querystringsafe_base64.decode(base64dotted_str) == decoded_str
+
+
+def test_fill_padding_unpadded():
+    """Check that fill_padding will fix padding."""
+    unpadded_string = 'MQ'
+    padded_string = querystringsafe_base64.fill_padding(unpadded_string)
```

Comment:
`assert querystringsafe_base64.fill_padding(unpadded_string) == 'MQ..'` should be ok: there is only one correct encoding for that string.